static/js/lib/ecommerce_test.js

```diff
@@ -40,6 +42,82 @@ describe("ecommerce", () => {
     })
   })
 
+  describe("calculateTax", () => {
+    it("calculates the tax of an item", () => {
+      const item = {
+        ...makeItem(),
+        price: "100"
+      }
+      const coupon = {
+        ...makeCouponSelection(item),
+        amount: "0"
+      }
+      assert.equal(calculateTax(item, coupon, 20), 20)
+    })
+
+    it("calculates the tax of an item including the 100% off coupon", () => {
+      const item = {
+        ...makeItem(),
+        price: "123"
+      }
+      const coupon = {
+        ...makeCouponSelection(item),
+        amount: "1"
+      }
+      assert.equal(calculateTax(item, coupon, 20), 0)
+    })
+
+    it("calculates the tax of an item including the 50% off coupon", () => {
+      const item = {
+        ...makeItem(),
+        price: "200"
+      }
+      const coupon = {
+        ...makeCouponSelection(item),
+        amount: "0.5"
+      }
+      assert.equal(calculateTax(item, coupon, 20), 20)
+    })
+  })
+
+  describe("calculateTotalAfterTax", () => {
+    it("calculates the total including the tax for an item", () => {
```

Comment:
Same as above comment.

Reply:
Should get resolved in https://github.com/mitodl/mitxpro/pull/2779/commits/b3070a71d3d4fa1cc88cfffbd1bff09bf17a35b6